packages/framework/src/Resources/views/Admin/Form/fileuploadFields.html.twig

```diff
@@ -62,7 +74,7 @@
                         {{ 'Drag file here'|trans }}
                     </div>
                     <div class="box-upload__place__drop__info">
-                        {{ infoText|default('') }}
```

Comment:
You didn't update the usage, eg.: https://github.com/shopsys/shopsys/blob/d29a9e8719fcda7835b432ef1dab323c8d6692f0/packages/framework/src/Resources/views/Admin/Content/Brand/detail.html.twig#L35